.github/workflows/primer_run_main.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
         run: |
           . venv/bin/activate
           pip install -e .
-          python tests/primer/primer_tool.py run --type=main 2>warnings.txt
+          python tests/primer/primer_tool.py run --type=main --jobs=0 2>warnings.txt
```

Comment:
I think this is really dangerous and will make things a lot worse. We discussed it at https://github.com/PyCQA/pylint/pull/6723#discussion_r884134628.

Reply:
Yeah, I guess you're right.

Let's wait on your investigation before adding this.